[2023-03-22T04:04:14.154+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: CarFine.get_file scheduled__2023-03-22T03:55:00+00:00 [queued]>
[2023-03-22T04:04:14.163+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: CarFine.get_file scheduled__2023-03-22T03:55:00+00:00 [queued]>
[2023-03-22T04:04:14.163+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T04:04:14.163+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-03-22T04:04:14.163+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T04:04:14.177+0000] {taskinstance.py:1303} INFO - Executing <Task(BashOperator): get_file> on 2023-03-22 03:55:00+00:00
[2023-03-22T04:04:14.181+0000] {standard_task_runner.py:55} INFO - Started process 738 to run task
[2023-03-22T04:04:14.183+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'CarFine', 'get_file', 'scheduled__2023-03-22T03:55:00+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/CarFineDAG.py', '--cfg-path', '/tmp/tmpgkslwpv7']
[2023-03-22T04:04:14.183+0000] {standard_task_runner.py:83} INFO - Job 267: Subtask get_file
[2023-03-22T04:04:14.239+0000] {task_command.py:388} INFO - Running <TaskInstance: CarFine.get_file scheduled__2023-03-22T03:55:00+00:00 [running]> on host f331b8a34e72
[2023-03-22T04:04:14.290+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=CarFine
AIRFLOW_CTX_TASK_ID=get_file
AIRFLOW_CTX_EXECUTION_DATE=2023-03-22T03:55:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-22T03:55:00+00:00
[2023-03-22T04:04:14.291+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-03-22T04:04:14.291+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'wget https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json ']
[2023-03-22T04:04:14.300+0000] {subprocess.py:86} INFO - Output:
[2023-03-22T04:04:14.303+0000] {subprocess.py:93} INFO - --2023-03-22 04:04:14--  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:04:22.223+0000] {subprocess.py:93} INFO - Resolving www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)... 162.125.81.15, 2620:100:6035:15::a27d:550f
[2023-03-22T04:06:33.050+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:06:33.050+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:06:33.051+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:06:33.051+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:06:34.050+0000] {subprocess.py:93} INFO - --2023-03-22 04:06:34--  (try: 2)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:08:44.121+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:08:44.122+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:08:44.122+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:08:44.122+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:08:46.122+0000] {subprocess.py:93} INFO - --2023-03-22 04:08:46--  (try: 3)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:10:57.242+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:10:57.242+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:10:57.243+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:10:57.243+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:11:00.242+0000] {subprocess.py:93} INFO - --2023-03-22 04:11:00--  (try: 4)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:13:10.362+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:13:10.362+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:13:10.362+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:13:10.362+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:13:14.362+0000] {subprocess.py:93} INFO - --2023-03-22 04:13:14--  (try: 5)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:15:25.529+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:15:25.530+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:15:25.530+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:15:25.530+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:15:30.530+0000] {subprocess.py:93} INFO - --2023-03-22 04:15:30--  (try: 6)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:17:40.697+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:17:40.698+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:17:40.698+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:17:40.698+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:17:46.698+0000] {subprocess.py:93} INFO - --2023-03-22 04:17:46--  (try: 7)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:18:50.466+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... connected.
[2023-03-22T04:18:51.308+0000] {subprocess.py:93} INFO - HTTP request sent, awaiting response... 200 OK
[2023-03-22T04:18:51.309+0000] {subprocess.py:93} INFO - Length: 259778 (254K) [text/plain]
[2023-03-22T04:18:51.309+0000] {subprocess.py:93} INFO - Saving to: ‘data.json’
[2023-03-22T04:18:51.515+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:18:51.771+0000] {subprocess.py:93} INFO -      0K .......... .......... .......... .......... .......... 19%  108K 2s
[2023-03-22T04:18:51.883+0000] {subprocess.py:93} INFO -     50K .......... .......... .......... .......... .......... 39%  445K 1s
[2023-03-22T04:18:51.931+0000] {subprocess.py:93} INFO -    100K .......... .......... .......... .......... .......... 59% 1.03M 0s
[2023-03-22T04:18:51.992+0000] {subprocess.py:93} INFO -    150K .......... .......... .......... .......... .......... 78%  819K 0s
[2023-03-22T04:18:52.031+0000] {subprocess.py:93} INFO -    200K .......... .......... .......... .......... .......... 98% 1.23M 0s
[2023-03-22T04:18:52.032+0000] {subprocess.py:93} INFO -    250K ...                                                   100% 6.87T=0.7s
[2023-03-22T04:18:52.032+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:18:52.033+0000] {subprocess.py:93} INFO - 2023-03-22 04:18:52 (351 KB/s) - ‘data.json’ saved [259778/259778]
[2023-03-22T04:18:52.033+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:18:52.034+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-03-22T04:18:52.056+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=CarFine, task_id=get_file, execution_date=20230322T035500, start_date=20230322T040414, end_date=20230322T041852
[2023-03-22T04:18:52.104+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-03-22T04:18:52.161+0000] {taskinstance.py:2585} INFO - 1 downstream tasks scheduled from follow-on schedule check
